todoTask/todoTask.js

```diff
@@ -1,85 +1,59 @@
-﻿// ------------------------global varibals--------------------
```

Comment:
1. It's not recommended to add "nice" symbols to comments. Keep it as simple as possible
2. Don't comment obvious things. Comments should be used only for intentionally strange code, or for high-level review